reporting.md

```diff
@@ -618,15 +636,14 @@ In the event of a delivery failure, regardless of the delivery method, a
 sender SHOULD attempt redelivery for up to 24hrs after the initial
 attempt.  As previously stated the reports are optional, so while it is
 ideal to attempt redelivery, it is not required.  If multiple retries
-are attempted, ideally they would be on a logarithmic scale.
+are attempted, ideally they would be on an exponential scale.
```

Comment:
"If multiple retries are attempted, they SHOULD be done with exponential backoff."

?